### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by ericsomto** [mwageringel/everest#23]
There should be a detailed help feature that should list how to play the game

**Comment by HalosGhost** [comment]
Maybe a good middle-ground would be to offer a summary of the rule(s) introduced in each level after the level is complete. This both offers confirmation to users that they understood correctly, and offers a means of taking a break but being able to dive back in somewhat quickly.

**Comment by sbliven** [comment]
Yeah, some sort of hint system would be nice. Maybe new operators could have some examples. I found myself brute forcing level 8 because I hadn't built up a good definition for the dotted | operator.
